src/main/java/com/bringframework/exceptions/NoSuchBeanException.java

```diff
@@ -0,0 +1,8 @@
+package com.bringframework.exceptions;
+
+/**
+ * Exception thrown when no beans are found.
+ */
+public class NoSuchBeanException extends RuntimeException {
```

Comment:
Maybe not as a part of this issue: exceptions MUST provide more info about what bean was not founf
The same goes for NoUniqueBeanException: provide what type and name of bean is not unique

Reply:
I think we will add more after identifying all the places where it will be used. 
And thanks for the comment.